docs/source/basics/local_development.md

```diff
@@ -137,40 +73,29 @@ wf.assembly_task(
 )
 ```
 
-Running this script will run `assembly_task`, just the same as the previous `run` command. You can add to this though,
-and run multiple tasks in the same script, as below.
-
-```python
-# filename: test.py
-
-from latch.types import LatchFile
-
-import wf
+You can execute the script in `latch develop` like so:
 
-sam = wf.assembly_task(
-    read1=LatchFile("latch:///read1.txt"),
-    read2=LatchFile("latch:///read2.txt"),
-)
-
-wf.sort_bam_task(sam=sam)
+```console
+>>> python3 scripts/test_task.py
 ```
 
-Hence using scripts, you can rapidly test the interactions between tasks, increasing our speed of workflow development.
+You can execute any python code in your workflow environment. Functions and scripts do not have to be latch specific so you can test library code or binaries as well. Think of the environment as a snapshot of the computer your tasks run on.
 
-## Exploring the environment
+## Notes on the test environment
 
-If you are running into issues with your environment (say, for example, a binary isn't where you expect it to be), you can see for yourself exactly what the environment looks like. Simply run `shell` in the REPL to get a fully functional bash session in the Docker container you created.
+It is important that any changes to the code are done on your local machine -- these changes will be synced into the latch develop environment and saved on your local computer as well. Changes made directly in the latch develop environment are not saved and are not synced back to your local computer. Moreover, they may be overwritten by the sync process.
 
-```console
->>> shell
+We use `rsync` to bring changes from your local workflow directory to the latch develop environment. We recursively copy changes to the `/root` directory in the development environment. For example, this line in the default docker image creates the `wf` directory in the cloud environment:
```

Comment:
mentioning dockerfile modification here makes it seem like `rsync` is not automatic